### PR TITLE
apt: add Malayalam translation

### PR DESCRIPTION
**Pull request by avronr**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ml/linux/apt.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# apt
+
+> ഡെബിയൻ അടിസ്ഥാനമാക്കിയുള്ള വിതരണങ്ങൾക്കായുള്ള പാക്കേജ് മാനേജുമെന്റ് യൂട്ടിലിറ്റി.
+> ഉബുണ്ടു പതിപ്പുകളിൽ 16.04ലും അതിനുശേഷമുള്ളതിലും സംവേദനാത്മകമായി ഉപയോഗിക്കുമ്പോൾ apt-get പകരം വയ്ക്കാൻ ശുപാർശ ചെയ്യുന്നതു.
+
+- ലഭ്യമായ പാക്കേജുകളുടെയും പതിപ്പുകളുടെയും പട്ടിക അപ്‌ഡേറ്റുചെയ്യുക (മറ്റ് ʻapt` കമാൻഡുകൾക്ക് മുമ്പ് ഇത് പ്രവർത്തിപ്പിക്കാൻ ശുപാർശ ചെയ്യുന്നു):
```
Comment (navarroaxel) [inline pages.ml/linux/apt.md:6]: the backtick here:
```suggestion
- ലഭ്യമായ പാക്കേജുകളുടെയും പതിപ്പുകളുടെയും പട്ടിക അപ്‌ഡേറ്റുചെയ്യുക (മറ്റ് `apt` കമാൻഡുകൾക്ക് മുമ്പ് ഇത് പ്രവർത്തിപ്പിക്കാൻ ശുപാർശ ചെയ്യുന്നു):
```

File: pages.ml/linux/apt.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- ഒരു പാക്കേജിന്റെ വിവരങ്ങൾ കാണിക്കുക:
+
+`apt show {{പാക്കേജ്}}`
+
+- ഒരു പാക്കേജ് ഇൻസ്റ്റാൾ ചെയ്യുക, അല്ലെങ്കിൽ ലഭ്യമായ ഏറ്റവും പുതിയ പതിപ്പിലേക്ക് അപ്‌ഡേറ്റ് ചെയ്യുക:
+
+`sudo apt install {{പാക്കേജ്}}`
+
+- ഒരു പാക്കേജ് നീക്കംചെയ്യുക (പകരം `purge' ഉപയോഗിക്കുന്നത് അതിന്റെ കോൺഫിഗറേഷൻ ഫയലുകളും നീക്കംചെയ്യുന്നു):
```
Comment (navarroaxel) [inline pages.ml/linux/apt.md:22]: the backtick here:
```suggestion
- ഒരു പാക്കേജ് നീക്കംചെയ്യുക (പകരം `purge` ഉപയോഗിക്കുന്നത് അതിന്റെ കോൺഫിഗറേഷൻ ഫയലുകളും നീക്കംചെയ്യുന്നു):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @avronr, this is our first Malayalam translation :tada: 

I don't speak the language but just 2 backticks to fix.

**Comment by avronr** [comment]
@navarroaxel thanks for reviewing my PR.
I've made committed your suggestions. :)

**Review by navarroaxel — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]

**Pull request merged**